### PR TITLE
Translations update from Hosted Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [PolyMC/PolyMC](https://hosted.weblate.org/projects/polymc/polymc/).

Current translation status:

Weblate translation status

### CODE REVIEW

File: pl.po [hunk 1]
```diff
@@ -405,6 +406,10 @@ msgid ""
 "\n"
 "Please update both your operating system and PolyMC."
 msgstr ""
+"Konta Microsoft są jedynie dostępna na systemach macOS 10.13 lub nowszych, "
```
Comment (K0-RR) [inline pl.po]: "jedynie dostępna"...

File: pl.po [hunk 2]
```diff
@@ -1923,6 +1928,10 @@ msgid ""
 "\n"
 "Do you want to play the demo?"
 msgstr ""
+"Te konto nie posiada zakupionego Minecrafta.\n"
```
Comment (K0-RR) [inline pl.po:1931]: And another one... "Te konto"

**Comment by Scrumplex** [comment]
@RDKRACZ Is something wrong in the translations? If so, you can fix it on Weblate

**Comment by K0-RR** [comment]
@Scrumplex those are spelling errors made by someone who didn't read even once what they have typed.
 
What does "&" mean and why it's inside a word? I have seen it a few times already and not all translations contain it.
Screenshot from 2022-03-14 13-33-25

**Comment by Scrumplex** [comment]
Not sure, it's also present in the source code: https://github.com/PolyMC/PolyMC/blob/develop/launcher/ui/widgets/CustomCommands.ui#L32

**Pull request merged**